viewing_party/main.py

```diff
@@ -0,0 +1,240 @@
+# Wave 1
+def create_movie(title, genre, rating):
+    # if bool(title) == True and bool(genre) == True and bool(rating) == True:
+    if title and genre and rating:
+        movie_dict = {
+            "title": title,
+            "genre": genre,
+            "rating": rating
+        }
+        return movie_dict
+    else:
+        return None
+
+def add_to_watched(user_data, movie):
+    user_data = {
+        "watched": []
+    }
+    user_data["watched"].append(movie)
+    return user_data
+
+def add_to_watchlist(user_data, movie):
+    user_data = {
+        "watchlist": []
+    }
+    user_data["watchlist"].append(movie)
+    return user_data
```

Comment:
These two functions are overwriting the parameter `user_data` and then modifying the version set up in the function rather than the version being passed in.  Consider what would happen if either function is called with `user_data` that has lists of movies in the "watched" and "watchlist" lists.  Rather than returning a version of the original `user_data` with the new movie added appropriately, this will return a dictionary with one key and the value is a list of one element.